project-keeper/src/main/java/com/exasol/projectkeeper/validators/files/FileTemplateFromResource.java

```diff
@@ -41,14 +41,20 @@ public FileTemplateFromResource(final String templateResource, final String path
     }
 
     /**
-     * Add a replacement definition. {@link FileTemplateFromResource} identifies variables in the template by prefix "$"
-     * and replaces each variable with the value provided beforehand.
+     * Add a replacement definition. {@link FileTemplateFromResource} identifies variables in the template by prefix
+     * {@code $} and replaces each variable with the value provided beforehand.
```

Comment:
If I understand correctly, then the curly braces in `if: ${{ $freeDiskSpace }}` will remain unreplaced.
Is this OK (for GH workflows) ?

Reply:
Yes, that's OK